packages/snaps-simulation/src/request.ts

```diff
@@ -106,6 +117,8 @@ export function handleRequest({
             result: getSafeJson(result),
           },
           notifications,
+          errors,
+          events,
```

Comment:
Hmm, wondering if these could be confused for thrown errors / emitted events instead of values from `snap_track...`

Reply:
Can rename to `trackedErrors` and `trackedEvents`? Or nest them in a `tracked` property or whatever.